### PR TITLE
Z index faker image

### PR DESCRIPTION
**Pull request by prawnto**
- z-index should no longer have conflicts
- faker doesn't generate multiple random images, have switched to picsum and applied it for LISTINGS. if ok, i will do so for booking as well

### CODE REVIEW

**Comment by prawnto** [comment]
linked faker image to booking, standardised booking views, added faker location. 

bookings links TBC

**Review by goh0222 — APPROVED** [review]

File: app/assets/stylesheets/components/_footer.scss [hunk 1]
```diff
@@ -9,7 +9,7 @@
   width: 100%;
   padding: 0px 0px;
   color: rgba(0,0,0,0.3);
-  z-index: 99;
+  z-index: 100;
   color: $gray;
 }
```
Comment (goh0222) [inline app/assets/stylesheets/components/_footer.scss:13]: remove line 11, as we have line 13

File: app/views/bookings/show.html.erb [hunk 2]
```diff
@@ -1,14 +1,41 @@
...
+<%# <div class="page__bar"></div><br> %>
+<%# <h1>Number of guests: <%= @booking.num_of_guests %></h1> %>
+<%# <h1>Check in: <%= @booking.start_date %></h1> %>
+<%# <h1>Check out: <%= @booking.end_date %></h1> %>
+
+<%# <div class="booking__show"> %>
+  <%# <%= link_to "Show all my bookings", bookings_path %> %>
+  <%# <%= link_to "Add review", new_listing_review_path(@listing) %> %>
+  <%# <%= button_to "Cancel reservation", @booking, method: :delete, class: "btn btn-danger" %> %>
+<%# </div> %>
```
Comment (goh0222) [inline app/views/bookings/show.html.erb]: remove all the commented lines

File: app/views/listings/index.html.erb [hunk 3]
```diff
@@ -2,16 +2,22 @@
...
+            <div class="card-trip">
+              <%= image_tag(listing.image_url) %>
+              <h2><%= listing.title %></h2>
+            </div>
+          <% end %>
+        <% end %>
       </div>
+    <% else %>
+      <h2>No listings made yet leh</h2>
+      <%= render "shared/search" %>
```
Comment (goh0222) [inline app/views/listings/index.html.erb]: line 19 seems not applicable for host

File: app/views/listings/show.html.erb [hunk 4]
```diff
@@ -3,12 +3,8 @@
 
   <%= render @listing %>
 
-<div class="btn btn-grab">
-  <%= link_to "Grab now", new_listing_booking_path(@listing) %>
-</div>
-
   <div class="card-trip">
-    <%= image_tag("house1.jpeg", class: "d-block w-100 page__listing") %>
+    <%= image_tag(@listing.image_url) %>
```
Comment (goh0222) [inline app/views/listings/show.html.erb:7]: yeah!

File: app/assets/stylesheets/components/_footer.scss (hunk 1 above)
Comment (prawnto) [reply app/assets/stylesheets/components/_footer.scss:13]: ok

File: app/views/bookings/show.html.erb (hunk 2 above)
Comment (prawnto) [reply app/views/bookings/show.html.erb]: ok

File: app/views/listings/index.html.erb (hunk 3 above)
Comment (prawnto) [reply app/views/listings/index.html.erb]: ok

**Comment by prawnto** [comment]
have amended based on given feedback, pls review!

**Pull request merged**